### PR TITLE
data:i-am-amrendra.json

### PR DESCRIPTION
**Pull request by i-am-amrendra**

### CODE REVIEW

File: data/Amrendra_Singh.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Amrendra Snigh",
```
Comment (ChinmayMhatre) [inline data/Amrendra_Singh.json:2]:
```suggestion
  "name": "Amrendra Singh",
```
Maybe name was misspelled

File: data/Amrendra_Singh.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer, DSA Enthusiastic and DevOps Learner",
+  "avatar": "https://github.com/account",
```
Comment (ChinmayMhatre) [inline data/Amrendra_Singh.json:6]:
```suggestion
  "avatar": "https://github.com/i-am-amrendra.png",
```
please use this url

File: data/Amrendra_Singh.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+       "name": "Medium",
+       "url": "https://medium.com/@Amrendra_Singh/"
+       "icon": "FaMedium"
+    },
+    {
+      "name": "Stackoverflow"
+      "url": "https://stackoverflow.com/users/20550455/amrendra-singh"
+      "icon": "FaStackoverflow"
```
Comment (ChinmayMhatre) [inline data/Amrendra_Singh.json:31]:
```suggestion
      "icon": "FaStackOverflow"
```
Icon spelling error

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Please rename the json file to your GitHub username

**Review by ChinmayMhatre — APPROVED** [review]

File: data/i-am-amrendra.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+       "icon": "FaGithub"
+    },
+    {
+       "name": "Instagram",
+       "url":  "https://instagram.com/amrendra_xsingh",
+       "icon": "FaInstagram"
+    },
+    {
+       "name": "Medium",
+       "url": "https://medium.com/@Amrendra_Singh/"
```
Comment (ChinmayMhatre) [inline data/i-am-amrendra.json:25]:
```suggestion
       "url": "https://medium.com/@Amrendra_Singh/",
```
missing comma

File: data/i-am-amrendra.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+       "icon": "FaMedium"
+    },
+    {
+      "name": "StackOverflow"
```
Comment (ChinmayMhatre) [inline data/i-am-amrendra.json:29]: missing comma
```suggestion
      "name": "StackOverflow",
```

File: data/i-am-amrendra.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      "url": "https://stackoverflow.com/users/20550455/amrendra-singh"
```
Comment (ChinmayMhatre) [inline data/i-am-amrendra.json:30]:
```suggestion
      "url": "https://stackoverflow.com/users/20550455/amrendra-singh",
```
missing comma

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
looks good only a few missing commas

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Comment by ChinmayMhatre** [comment]
@Pradumnasaraf  I think one of the commits had a failed build before you merged it 
image

**Comment by Pradumnasaraf** [comment]
> @Pradumnasaraf  I think one of the commits had a failed action before you merged it 
> image
> 

Yep, I did that mistake 🙁. That caused the action to fail on other PRs. No worries, everything is sorted out now.

**Comment by i-am-amrendra** [comment]
Thank u very much @eddiejaoude @ChinmayMhatre @Pradumnasaraf  for helping me out.🫡

**Pull request merged**